### PR TITLE
The compatible info only show for installed app

### LINKED ISSUE #100: The app store displays confusing incompatibility warning during app installation
# Observed behavior

When installing a new app, the app store will show a confusing warning:

> The installed version of this app is not compatible with this AiiDAlab environment.

See also screenshot:

Screen Shot 2022-06-28 at 12 38 38

# Expected behavior

The app store should not check compatibility during the installation, but only after the installation has completed.

### PR DESCRIPTION
**Pull request by unkcpz**
Fixes #100 

The compatible info will be shown during the installation, which is not the expected behavior. The compatible info is only shown for the app installed.

### CODE REVIEW

**Comment by danielhollas** [comment]
@unkcpz could you please provide screenshots so we know what this PR is fixing. I am not that familiar with this functionality so screenshots always help a lot.

Otherwise this looks good to me.

**Comment by unkcpz** [comment]
This PR should go after https://github.com/aiidalab/aiidalab/pull/349, but I just find there is an unexpected behavior of my implementation. I'll summarize both with screenshots. I make it a draft at the moment.

**Comment by unkcpz** [comment]
Peek 2023-02-09 11-08

Even after the fix of https://github.com/aiidalab/aiidalab/pull/349, from the screenshot you can see during the installation the yellow warning box showing "Reason for incompatibility" will bleak, which should not the case. It is caused from the box is pop out without checking the app is installed.

**Review by danielhollas — APPROVED** [review]
Thanks for additional details @unkcpz!

This change looks good to me. I think it is hard to test in isolation, so I would propose to merge this one first, to make it easier to test the other PRs.

These PRs for `aiidalab` and `aiidalab-home` are imho quite important so would be good to always get review by both me and @yakutovicha

**Comment by yakutovicha** [comment]
I just made the PR `blocked` till we decide whether to go with the sandbox approach or not.

**Comment by unkcpz** [comment]
After https://github.com/aiidalab/aiidalab/pull/357 change, we are ready with this PR.

**Comment by unkcpz** [comment]
Sorry, this PR is not needed I think it is all fixed by https://github.com/aiidalab/aiidalab/pull/357. I'll give it a test.

**Comment by unkcpz** [comment]
Just did the check, this PR is still needed. The changes applied will show the `compatibility_info` only for the installed app. 

@danielhollas can you give this a second look? I think don't need to test, I test it and I am confident with the issue I show inhttps://github.com/aiidalab/aiidalab-home/pull/131#issuecomment-1423938062 is gone.

**Review by danielhollas — APPROVED** [review]
LGTM :rocket: Thanks @unkcpz, good to have this sorted out. Please, after you merge this, merge main branch into #130

**Pull request merged**
